src/scsiencrypt.cpp

```diff
@@ -314,7 +314,7 @@ bool SCSIExecute(const std::string& tapedrive, unsigned char *cmd_p, int cmd_len
   std::cout << "\n";
 
   std::cout << "SCSI Data: ";
-  for (int i = 0; i < dxfer_len; i++) {
+  for (int i = 0; i < 4 + BSSHORT(dxfer_p + 2); i++) {
```

Comment:
this line is difficult to read. Can we simplify it?

Reply:
This is some assistive debug output I forgot to strip out - I will push a new revision that backs this out.